public/app/types/folders.ts

```diff
@@ -23,7 +23,7 @@ export interface FolderState {
 }
 
 export interface FolderInfo {
-  id: number;
-  title: string;
-  url: string;
+  id: number | undefined;
+  title: string | undefined;
+  url: string | undefined;
```

Comment:
nit: why not use `?`
```suggestion
  id?: number;
  title?: string;
  url?: string;
```